code/Model/Recipient/EmailRecipient.php

```diff
@@ -151,7 +152,8 @@ protected function getFormParent()
 
         $formID = Controller::curr()->getRequest()->getSession()->get($sessionNamespace . '.currentPage');
         if ($formID) {
-            return UserDefinedForm::get()->byID($formID);
+            $parent_class = get_class(Injector::inst()->get(UserDefinedForm::class));
```

Comment:
Is the `get_class()` necessary?  I might be werong, though it seems like it would work fine without get_class and we could change $parent_class to simply $parent.  Id's expect that to work slightly better with type hinting in IDE's